docs/reference/commandline/build.md

```diff
@@ -28,6 +28,8 @@ Options:
       --cpu-period int          Limit the CPU CFS (Completely Fair Scheduler) period
       --cpu-quota int           Limit the CPU CFS (Completely Fair Scheduler) quota
   -c, --cpu-shares int          CPU shares (relative weight)
+      --cpu-rt-period=0         Limit the CPU real-time period in microseconds
+      --cpu-rt-runtime=0        Limit the CPU real-time runtime in microseconds
```

Comment:
I think this doesn't match the actual output of the usage output; defaults are now shown in the description since we switched to "Cobra" for the CLI